service/src/test/java/eu/europa/ec/fisheries/uvms/user/service/arquillian/TransactionalTests.java

```diff
@@ -0,0 +1,35 @@
+package eu.europa.ec.fisheries.uvms.user.service.arquillian;
+
+import org.junit.After;
+import org.junit.Before;
+
+import javax.inject.Inject;
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.transaction.HeuristicMixedException;
+import javax.transaction.HeuristicRollbackException;
+import javax.transaction.NotSupportedException;
+import javax.transaction.RollbackException;
+import javax.transaction.SystemException;
+import javax.transaction.UserTransaction;
+
+public class TransactionalTests extends BuildUserServiceDeployment {
```

Comment:
What is the purpose of this file, is there any Arquillian test? Keep all Arquillian-related artifacts in a separate commit in a separate branch in case we need them.